expcore/gui.lua

```diff
@@ -496,19 +496,56 @@ end)
 
 ]]
 function Gui.left_toolbar_button(sprite,tooltip,element_define,authenticator)
-    return Gui.element{
+    local button = Gui.element{
         type = 'sprite-button',
         sprite = sprite,
         tooltip = tooltip,
         style = Gui.top_flow_button_style
     }
     :style{
+        minimal_width = 36,
+        height = 36,
         padding = -2
     }
     :add_to_top_flow(authenticator)
     :on_click(function(player,_,_)
         Gui.toggle_left_element(player, element_define)
     end)
+    element_define.toolbar_button = button.name
+    return button
+end
+
+--[[-- Styles the left toolbar button according to the state given
+@tparam LuaPlayer player the player to hide the elements for
+@tparam table element_define the element that you want to toggle for the player
+@tparam[opt] boolean state if given then the state will be set to this state
+@treturn boolean the new visible state of the element
```

Comment:
ln519 you are not hiding any elements in this function
ln520 the element is not being toggled the toolbar button is
ln522 no value is being returned by the function